### DISCUSSION TITLE
Letras Parte 3

### DISCUSSION THREAD

**Discussion by Consolo0** [IIC2133-PUC/2024-1#158] [Tarea 2] [answered]
Hola, en la parte 3 hay un input el cual se reciba la letra "ñ" o solo son letras que se conforman del abecedario inglés

**Comment by sebacarrascop** [comment, marked as answer]
hola!, usamos alfabeto ingles para la generación de test, asi que puedes asumir que no habrá ninguna ñ.

**Reply by Consolo0** [reply]
vale gracias!
